src/esp/assets/attributes/ObjectAttributes.cpp

```diff
@@ -17,6 +17,38 @@ const std::map<std::string, esp::assets::AssetType>
         {"suncg", AssetType::SUNCG_SCENE},
 };
 
+const std::string AbstractObjectAttributes::JSONConfigExample =
```

Comment:
Compiling this into the source code is weird IMO -- it seems like it will be hard to discover. If the goal is to provide users with an example of these, files in the examples folder makes a lot more sense.

Reply:
I'm going to link it to the docs directly, and I did not want to introduce more files that would break tests if they were not around. :shrug:

Reply:
I was trying to kill 2 birds with one stone here, and perhaps I shouldn't be.  On the one hand, i want an easily maintained and obvious construction to use to test JSON parsing within the attributes managers (which I feel is better served with an internal static string within the attributes that can be modified when the class itself is extended). 

On the other hand, I wanted to provide a "json example" using the tags for documentation/.rst, while avoiding adding external files, to minimize the required support in the future for this functionality.

Reply:
Yeah, I am not sure if a string in the source code makes a good example.  You either have to load the python module and print the string or piece it together from the source to make a file to be loaded.

Reply:
If it's just about showing that in the docs, you can reference this string using the `@snippet` Doxygen command (if you wrap it in some delimiters), or with more flexibility using `.. include::` in Python docs similarly as you did in Python tutorials. No need to put that in a dedicated file.

Reply:
I agree that files make sense for good examples, but we don't want users to accidentally find counter examples in our data folder, so it makes sense to hide those the code. We can always ref these into the doc rst files if we want to expose them.

Reply:
So how about I make good example files, and these weird "examples" i rename to be test-related, and remove the python bindings (since users would never really want to see these anyway)?